en/customization.md

```diff
@@ -87,7 +87,7 @@ You can set the value of the `math` attribute in the Front Matter of the file in
 
 ## Comment
 
-Eureka supports using Disqus, Utterances and Commento as comment system。If you need to configure comment, you just need to configure the `comment` in `config/_default/params.yaml`
+Eureka supports using Disqus, Utterances, Valine and Commento as comment system。If you need to configure comment, you just need to configure the `comment` in `config/_default/params.yaml`
```

Comment:
Please add the detail explanation of valine. You can refer to `comment.utterances` or `diagram.mermaid` for example.